xdg-terminal-exec

```diff
@@ -175,55 +176,49 @@ save_cache() {
 	fi
 }
 
-# Parse all config files and populate $ENTRY_IDS with read desktop entry IDs
+# Parse all config files and populate $READ_ENTRY_IDS with read desktop entry IDs
 read_config_paths() {
+	debug '' '    > reading terminal list files:'
 	# All config files are read immediatelly, rather than on demand, even if it's more IO intensive
 	# This way all IDs are already known, and in order of preference, before iterating over them
 	IFS=':'
 	for config_path in ${CONFIGS}; do
-		debug "reading config '$config_path'"
 		# Nonexistant file is not an error
 		[ -f "$config_path" ] || continue
-		# Let `read` trim leading/trailing whitespace from the line
-		while IFS="$OIFS" read -r line; do
-			#debug "read line '$line'"
-			case $line in
-
-			# Catch directives first
-
-			# cache control
-			/enable_cache)
-				debug "found '$line' directive${XTE_CACHE_ENABLED+ (ignored)}"
+		debug '' "    $config_path"
+		# Let `read` trim leading/trailing whitespace from the IDs
+		while IFS=":$OIFS" read -r entry_id action_id; do
```

Comment:
this splits action not only on ':', but on whole original IFS, falsely consuming invalid lines